</br>
</br>
</br>
<details>
<summary><b>Establishing bioremediation potential of urban estuaries</b></summary>
<p>Sediments in the Great Bay Estuary of New Hampshire contain many pollutants that are detrimental to estuarine and human health. Mercury and polyaromatic hydrocarbons are a particular concern in sediment and sediment-associated biota while numerous wastewater treatment plants provide a source of antibiotics to the system. To assess the bioremediation potential of microbial communities found in the estuary the metagenomes of a contaminated site, Cocheco, and a pristine site, Oyster, were sequenced using Illumina TruSeq technology. The objectives of the study are to determine the microbial community profile in both sites and establish which taxa differ. A functional gene profile from both sites is established and presented the differences between the sites at a pathway level with emphasis on genes important in bioremediation (those associated with alkane, PAH and mercury metabolism, as well as antibiotic resistance). Taxonomic information was assigned by mapping the reads against NCBI Refseq, SILVA 16S and 18S rRNA, and Uniprot90 protein sequence databases. Additionally, an LCA algorithm was used to come up with a holistic taxanomical profiling. Based on the results we concluded that both sites were dominated by Proteobacteria although the relative abundance of classes differed. mer (mercury resistance) genes were identified at both sites. Genes associated with alkane, PAH and antibiotic degradation were also present in both data sets at similar levels. Overall these results indicate that although different estuarine sediments harbor taxonomically distinct bacterial communities the presence of functional genes relevant to bioremediation can be quite similar, suggesting whole metagenome sequencing approaches of the total community may be more informative than targeted 16S rRNA-based approaches when considering the bioremediation potential of a site.</p>
</details>
<details>
<summary><b>Aphaenogaster transcriptome analysis</b></summary>
<p>Thermal tolerance is a key trait governing the distributions of species, their abundances, and underlies their future responses to climate change. However, the genetic basis of thermal tolerance is poorly understood for most species. In this study, we present a novel regression approach to characterize the thermal reaction norm for all genes in an organism’s transcriptome. We apply this approach to parapatric ant species in the eastern United States; A. carolinensis and A. picea, which have warmer and colder distributions, respectively. Using this approach, we find that at least 10% of genes are thermally responsive. In both species, more genes are up-regulated in response to cold than warm temperatures, suggesting that there is less adaptive potential to heat than cold stress. Between species, A. carolinensis shuts down more genes in response to thermal extremes, whereas A. picea activates more genes in response to cold and hot temperatures. The genes that are shut down in A. carolinensis but up-regulated in A. picea included many metabolism, cellular organization and repair gene ontology terms. We conclude that these species have different mechanisms for responding to thermal stress. A. picea resists thermal shock by synthesizing heat shock and other protective molecules to maintain function, whereas A. carolinensis slows down metabolism and relies upon other mechanisms to tolerate thermal stress. In summary, we bring reaction norms into the genomic era with this study. Our results demonstrate how we can gain insight on the genetic basis of local adaption to climate using this approach.</p>
</details>
<details>
<summary><b>USGS Metagenomics analysis</b></summary>
<p>Airfields and aircrafts undergo routine deicing measures to enhance airtravel safety in cold weather conditions. Most of the chemicals found in deicing material used enter into surface water systems or ground water systems. This creates challenging environment to maintain healthy aquatic communities. In waters receiving organic pollution, the presence of biofilms dominated by bacteria in the Sphaerotilus-Leptothrix group is common and has been noted for over two centuries. The predominance of these bacteria over algae has been shown to depend upon the presence of and abundance of readily degradable substrates. The main aim of this study was to bring next generation sequencing techniques to bear on the study of these biofilms in an effort to answer two primary questions, a) to determine the influence of environmental variables on biofilm community structure across temporal and spatial gradients and b) to determine the environmental trigger(s) responsible for Sphaerotilus/Leptothrix sheath formation in these settings.</p>
</details>
<details>
<summary><b>Metatranscriptomic insights of bacterial populations in different digesters</b></summary>
<p>The main aim of this study was to get an insight into differential expression profiling of proteins across 3 different digester samples, BS, CHA and GMB. The filtered reads from all samples are combined and run through IDBA software to generate denovo metatranscriptomic assembly. The generated denovo metatranscriptomic assembly is subjected to gene finding software, MetaGeneMark, to obtain coding sequences from the assembly. The coding sequences obtained from MetaGeneMark are blasted against ncbi-nr protein database for annotation purposes. Raw reads are aligned to denovo metatranscriptomic assembly using bowtie for each sample separately. The raw counts obtained are run through RSEM and Trinity DE suite of scripts to find out differentially expressed proteins across samples and also to cluster the samples based on this information.</p>
</details>
<details>
<summary><b>454 Metagenomics analysis of NorthEastern Lakes</b></summary>
<p> </p>
</details>
<details>
<summary><b>CloVR</b></summary>
<p>Worked in developing and integrating bioinformatics pipelines using cloud computing and virtual machines. Also, built high end web applications using Jquery, ExtJs, and Perl-CGI. This work is done at Institute for Genome Sciences, Baltimore, MD as Bioinformatics Software Engineer, 2010-2012. More information regarding this project can be found <a href="http://clovr.org" target="_blank">here</a>.</p>
</details>
<details>
<summary><b>Development of a software tool for Metagenomic Analysis</b></summary>
<p>ARNAC (an Alignment based 16s-RNA Classifier) is developed for metagenomic data analysis using Object Oriented Perl. This is done as part of a coursework towards Professional Science Masters Degree in Bioinformatics (Genomics track) at Virginia Commonwealth University, 2010.</p>
</details>
<details>
<summary><b>Database Design for Restaurant Business</b></summary>
<p>Designed and implemented a database at physical and logical levels on Restaurant Business using relational database concepts using MySQL. This is done as part of a coursework towards Professional Science Masters Degree in Bioinformatics (Genomics track) at Virginia Commonwealth University, 2010, under the guidance of <a href="http://www.people.vcu.edu/~lparker/" target="_blank">Dr. Lorraine Parker, Computer Science Dept.</a></p>
</details>
<details>
<summary><b>Modeling of Motifs and Biofunctions</b></summary>
<p>Using data mining of all motifs and their biological functions from BioMart and Gene Ontology databases and applying statistics methods such as “Random Forest” package in R, I tried to rank high confidence biofunctions to each motif. This is done as part of a coursework towards Professional Science Masters Degree in Bioinformatics (Genomics track) at Virginia Commonwealth University, 2009, under the guidance of <a href="http://www.people.vcu.edu/~bwindle/" target="_blank">Dr. Bradford E. Windle, Medicinal Chemistry Dept.</a></p>
</details>
<details>
<summary><b>Metagenomic profiling of microbes in James River</b></summary>
<p>Hands on learning of sample collection, 16s primer design, cell culturing, cloning, and NGS analysis of 454 16s sequence data to establish qualitative and quantitative bacterial phylogeny of James River. This is done as part of a coursework towards Professional Science Masters Degree in Bioinformatics (Genomics track) at Virginia Commonwealth University, 2009.</p>
</details>
<details>
<summary><b>Drosophila melanogaster genome search web application</b></summary>
<p>Developed both front and back ends of D. melanogaster genome search web app using MYSQL, CGI and Perl. This is done as part of a coursework towards Professional Science Masters Degree in Bioinformatics (Genomics track) at Virginia Commonwealth University, 2008.</p>
</details>
<details>
<summary><b>Exploring Early Events of Protein Aggregation in Alzheimer’s Disease using Molecular Dynamics Simulation Studies</b></summary>
<p>The disease is mainly caused by extracellular deposition of beta amyloid plaques. There are 2 different beta amyloid proteins that are observed in the plaques, AB42 and AB40. Beta amyloid with a length of 42 amino acids (AB42) causes significant damages than Beta amyloid with a length of 40 amino acids (AB40), due to it’s increased hydrophobic nature. It is believed that the conformational changes between AB42 and AB40 will shed insight into the molecular mechanism of the disease. For this reason, I worked on simulating the protein folding events at femto second time frame using Molecular Simulation Software “AMBER”. This is done as part of a coursework towards Masters Degree in Bioinformatics (BioPhysics track) at Madras University, 2005 - 2007.
</p>
</details>

